doc/Building.md

```diff
@@ -8,12 +8,12 @@ This document is intended for mediasoup developers.
 The `package.json` file in the main folder includes the following scripts:
 
 
-### `num run typescript:build`
+### `npm run typescript:build`
```

Comment:
Can you merge v3 into your branch first? This was already fixed.

Reply:
OK, I'll finish it.